automation: install dnf5 copr module for Rawhide

vbenes opened the pull request:
We need to have dnf5 copr plugin installed in Rawhide as dnf5 is the default there.

ffmancera reviewed (dismissed):
Thanks for the contribution, could you check the CI? This is breaking plenty of jobs.

vbenes committed:
automation: install dnf5 copr module for Rawhide

We need to have dnf5 copr plugin installed in Rawhide as dnf5 is
the default there.

Signed-off-by: Vladimír Beneš <vbenes@redhat.com>
